TRANSLATION_NOTES.asc

```diff
@@ -1,61 +1,54 @@
-== Hinweise zur deutschen Übersetzung
+= Hinweise zur deutschen Übersetzung
 
-<<<<<<< HEAD
-In diesem Dokument werden alle Punkte rund um die deutsche Übersetzung gesammelt. In diesem Dokument möchten wir festlegen, ob der Leser mit Du oder Sie angesprochen wird, ob der Erzähler die 1. Person Singular oder die 1. Person Plural verwendet, oder wie technische Fachbegriffe übersetzt werden sollen.
-=======
-After forking this repository to translate the work, this file is where the notes for coordinating the translation work would go.
-Things like standardizing on words and expressions so that the work is consistent or notes on how the contributing process is to be handled.
->>>>>>> 811dffebd4ff1348819f8acc70f954bd4ad65057
+In diesem Dokument werden alle Informationen rund um die deutsche Übersetzung gesammelt. Hiermit möchten wir beispielsweise festlegen, ob der Leser mit "Du" oder "Sie" angesprochen wird, ob der Erzähler die 1. Person Singular oder die 1. Person Plural verwendet oder wie technische Fachbegriffe übersetzt werden sollen.
 
-=== Übersetzungsfortschritt
+== Übersetzungsfortschritt
 
-Beim Übersetzen sollte zusätzlich zur eigentlichen Übersetzung der Fortschritt jedes Kapitels in der Datei `status.json` hinterlegt werden. Diese Prozentangabe wird auf verschiedenen Seiten verwendet, um die Leser über den Fortschritt der jeweiligen Übersetzung zu informieren. 
+Beim Übersetzen sollte, zusätzlich zur eigentlichen Übersetzung, der Fortschritt jedes Kapitels in der Datei link:./status.json[`status.json`] hinterlegt werden. Diese Prozentangabe wird auf verschiedenen Seiten verwendet, um die Leser über den Fortschritt der jeweiligen Übersetzung zu informieren.
 
-<<<<<<< HEAD
-=== Workflow Git
+== Workflow Git
 
-Wenn Sie an der deutschen Übersetzung mitarbeiten wollen, können Sie dazu ein Fork erstellen und in diesem weiterarbeiten.
+Wenn Sie an der deutschen Übersetzung mitarbeiten wollen, können Sie dazu ein link:https://git-scm.com/book/de/v1/Distribuierte-Arbeit-mit-Git-xxx-An-einem-Projekt-mitarbeiten#Kleine,-öffentliche-Projekte[Fork] erstellen und in diesem weiterarbeiten.
 
-* Bitte erstellen Sie einen Pull-Request, wenn Sie ein Arbeitspaket abgeschlossen haben. Bitte beschreiben Sie im Pull-Request was Ihr zu mergender Branch enthält (Neue Übersetzung, Korrektur, ...).
+* Bitte erstellen Sie erst einen Pull-Request, wenn Sie ein Arbeitspaket abgeschlossen haben. Bitte beschreiben Sie im Pull-Request, was Ihr zu mergender Branch enthält (neue Übersetzungen, Korrekturen, etc.).
```

Comment:
```suggestion
* Bitte erstellen Sie erst einen Pull-Request, wenn Sie ein Arbeitspaket abgeschlossen haben. Bitte beschreiben Sie im Pull-Request, was Ihr zu mergender Branch enthält (neue Übersetzungen, Korrekturen usw.).
```

`etc` or `usw`? Also, I don’t think we use a comma before this ellipsis abbreviation.